### ISSUE TITLE
Automatically run `cosign init` for keyless mode

### ISSUE THREAD

**Issue by priyawadhwa** [sigstore/cosign#604]
Verifying SCTs (https://github.com/sigstore/cosign/pull/600) relies on the user running `cosign init` first so that the CT log public key is on disk for verification

We can automatically run `cosign init` for a user if they haven't already done it before verification! Once this is done, we can consider failing if SCT verifications fails (right now we just log a warning and move on)

**Comment by dlorenc** [comment]
+1 to either option there Priya. We should embed enough stuff to either initialize the directory or verify in memory

**Comment by priyawadhwa** [comment]
sgtm! I'll probably switch back to embedding the ct log public key and verifying in memory then, so that we don't have to depend on GCS to get the public key

**Comment by asraa** [comment]
This is done.
